services/navet-ms/lambdas/pollUser.js

```diff
@@ -66,9 +66,8 @@ async function requestNavetUser(payload, params) {
   if (
     Folkbokforingspost.Sekretessmarkering === 'J' ||
     Folkbokforingspost.SkyddadFolkbokforing === 'J'
-  ) {
-    throwError(404);
-  }
+  )
+    throw 'Forbidden';
```

Comment:
Missing curly braces?

Reply:
I removed curly braces to follow the same "coding style" as the rest of the file. But i can add it again :P